Added features like Total Pull Requests, Total Branches, Total Commits, Total Contributors.

rdxmiles opened the pull request:
Added features like Total Pull Requests, Total Branches, Total Commits, Total Contributors.
[![usage](https://img.shields.io/badge/amDeC-amFOSS-blue)](https://amdec.amfoss.in/) [![usage](https://img.shields.io/badge/FOSS%40Amrita-amDeC%202019-yellowgreen)](https://github.com/amfoss) [![usage](https://img.shields.io/badge/amFOSS-%20-brightgreen)](https://github.com/amfoss) [![usage](https://img.shields.io/static/v1?label=Stardox&message=Python&color=brightgreen)](https://github.com/amfoss)

rdxmiles committed:
Update stardox.py

Added features like Total Pull Requests, Total Branches, Toatal Commits, Total Contributors.

pep8speaks commented:
Hello @bhavesh2699! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`src/stardox.py`](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py):

> [Line 244:13](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L244): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 244:17](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L244): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 245:21](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L245): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 245:26](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L245): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 248:15](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L248): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> [Line 253:9](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L253): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 256:13](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L256): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 258:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L258): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (92 > 79 characters)
> [Line 261:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L261): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (97 > 79 characters)
> [Line 261:89](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L261): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 262:1](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L262): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 263:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L263): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (89 > 79 characters)
> [Line 266:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L266): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (95 > 79 characters)
> [Line 266:87](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L266): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 267:1](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L267): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 268:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L268): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (103 > 79 characters)
> [Line 269:22](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L269): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> [Line 271:75](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L271): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 271:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L271): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)
> [Line 272:1](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L272): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 273:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L273): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (107 > 79 characters)
> [Line 274:26](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L274): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> [Line 276:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L276): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (91 > 79 characters)
> [Line 276:83](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L276): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 277:1](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L277): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 296:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L296): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (87 > 79 characters)
> [Line 297:27](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L297): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> [Line 299:77](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L299): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 299:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L299): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (85 > 79 characters)
> [Line 301:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L301): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (85 > 79 characters)
> [Line 302:29](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L302): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> [Line 304:74](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L304): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 304:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L304): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 306:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L306): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 307:30](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L307): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> [Line 309:74](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L309): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 309:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L309): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (82 > 79 characters)
> [Line 311:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L311): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (100 > 79 characters)
> [Line 312:35](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L312): [E225](https://duckduckgo.com/?q=pep8%20E225) missing whitespace around operator
> [Line 313:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L313): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)
> [Line 314:80](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L314): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (96 > 79 characters)
> [Line 314:88](https://github.com/0xPrateek/Stardox/blob/646f7afb8aeaf5d3dc93f6a70a6a6f305c1c9699/src/stardox.py#L314): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','



##### Comment last updated at 2020-01-20 17:11:25 UTC

rdxmiles committed:
Update stardox.py

Added features like Total Pull Requests, Total Branches, Total Commits, Total Contributors.

rdxmiles committed:
PEP 8 Correction

In the file src/stardox.py:
Line 256:80: E501 line too long (87 > 79 characters)
Line 257:27: E225 missing whitespace around operator
Line 258:80: E501 line too long (103 > 79 characters)
Line 258:91: E231 missing whitespace after ','
Line 258:95: E231 missing whitespace after ','
Line 259:80: E501 line too long (85 > 79 characters)
Line 260:29: E225 missing whitespace around operator
Line 261:80: E501 line too long (95 > 79 characters)
Line 261:83: E231 missing whitespace after ','
Line 261:87: E231 missing whitespace after ','
Line 262:80: E501 line too long (90 > 79 characters)
Line 263:30: E225 missing whitespace around operator
Line 264:80: E501 line too long (94 > 79 characters)
Line 264:82: E231 missing whitespace after ','
Line 264:86: E231 missing whitespace after ','
Line 265:80: E501 line too long (100 > 79 characters)
Line 266:35: E225 missing whitespace around operator
Line 267:80: E501 line too long (109 > 79 characters)
Line 267:97: E231 missing whitespace after ','
Line 267:101: E231 missing whitespace after ','

rdxmiles committed:
Update stardox.py

Added User Profile Section like Followers, Following, Stars, Repositories etc.
